scanner/config/config.go

```diff
@@ -171,6 +172,30 @@ type MatcherConfig struct {
 	VulnerabilityVersion string `yaml:"vulnerability_version"`
 }
 
+// resolveVersions returns values for ROX_VERSION and ROX_VULNERABILITY_VERSION
+// based on the current build information and version string name.  If the user
+// has explicitly set a VulnerabilityVersion in the configuration, it overrides
+// the default values to maintain backward compatibility with pre-existing
+// configs.
+func (c *MatcherConfig) resolveVersions() (roxVer, vulnVer string) {
+	roxVer = "dev"
+	vulnVer = "dev"
+	// Rely on buildinfo first to check release biulds, it is defined by
+	// build tags and has stronger reliability.
+	if buildinfo.ReleaseBuild || pkgversion.GetVersionKind(version.Version) == pkgversion.NightlyKind {
```

Comment:
nightlies are built with the `release` tag, so the nightly check may not be necessary. @BradLugo can you confirm?

Reply:
I worked under the assumption that they are not release builds. I will wait for the confirmation.

Reply:
Confirmed nightlies are built with the `release` tag. I removed the check.